staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -77,6 +77,10 @@ type Interface interface {
 
 	// WatchTracker provides the WatchTracker interface.
 	WatchTracker
+
+	// PriorityLevelMaxSeats returns a function that calculates maximum seats used by work estimator
+	// for a given priority level
+	PriorityLevelMaxSeats() fcrequest.MaxSeatsFunc
```

Comment:
I'm sorry, I guess I messed up my comment. I guess I didn't want the function that returns a function.
```
PriorityLevelMaxSeats(plName string) uint64
```

is what I wanted (I somehow though it was a config, not the interface...)

Reply:
makes sense, fwiw I like `PriorityLevelMaxSeats(plName string) uint64` too, I'll revert this change